tests/unit/test_retry.py

```diff
@@ -361,7 +362,7 @@ def test___call___and_execute_retry(self, sleep, uniform):
         assert on_error.call_count == 1
 
     # Make uniform return half of its maximum, which is the calculated sleep time.
```

Comment:
Update the comment here to remove halfing.

Reply:
Good catch! removed